PULL_REQUEST_TEMPLATE

```diff
@@ -20,5 +20,6 @@
 #### Screenshots (if appropriate)
 #### Checklist:
 - [ ] Documentation Updated
+- [ ] Change log Updated
```

Comment:
Can you make this a sub (tabbed) check? I don't imagine this would be checked if the Documentation wasn't updated.